src/app/containers/app/app.component.ts

```diff
@@ -278,15 +277,6 @@ export class AppComponent implements OnInit, OnDestroy {
     }
   }
 
-  // the fragment attribute in Angular is good however it only scrolls to the anchor tag
```

Comment:
I see you've removed this. I've just given it a try in Edge to make sure it wasn't needed for a different browser and it appears it was surplus to requirements!